localstack-core/localstack/utils/analytics/metadata.py

```diff
@@ -106,13 +106,15 @@ def get_machine_id() -> str:
     return doc["machine_id"]
 
 
-def get_localstack_edition() -> Optional[Literal["enterprise", "pro", "community"]]:
+def get_localstack_edition() -> Optional[Literal["enterprise", "pro", "community", "azure-alpha"]]:
     if os.path.exists("/usr/lib/localstack/.enterprise-version"):
         return "enterprise"
     elif os.path.exists("/usr/lib/localstack/.pro-version"):
         return "pro"
     elif os.path.exists("/usr/lib/localstack/.community-version"):
         return "community"
+    elif os.path.exists("/usr/lib/localstack/.azure-alpha-version"):
+        return "azure-alpha"
```

Comment:
could we maybe generalize this to simply check for `/usr/lib/localstack/.<x>-version` and return` `<x>`?

Reply:
Added this and a unit test for testing it.

Using `config.dirs.static_libs` (which should be `/usr/lib/localstack` in the container) now so that it can be tested more easily.